src/components/ButtonsList.tsx

```diff
@@ -0,0 +1,51 @@
+import { Button } from "./Button";
+
+type ButtonsListProps = {
+  filterHandler: (action: string) => void;
+  filter: string;
+  isReversed: boolean;
+};
+
+export const ButtonsList: React.FC<ButtonsListProps> = ({
+  filterHandler,
+  filter,
+  isReversed,
+}) => {
+  return (
+    <div className="buttons">
+      <Button
+        name="alphabet"
+        content="Sort alphabetically"
+        filterHandler={filterHandler}
+        filter={filter}
+        isReversed={isReversed}
+      />
```

Comment:
The `Button` component instances are missing the `key` prop which is necessary when rendering lists or collections in React to help the library identify which items have changed, are added, or are removed.